cmd/minishift/cmd/start.go

```diff
@@ -57,6 +57,7 @@ const (
 
 var (
 	dockerEnv        []string
+	dockerEngineOpt  []string
```

Comment:
@hferentschik This is auto picked up by libmachine and we don't need to change our provisioner for this.